utilities/transactions/pessimistic_transaction_db.h

```diff
@@ -97,14 +100,18 @@ class PessimisticTransactionDB : public TransactionDB {
   void GetAllPreparedTransactions(std::vector<Transaction*>* trans) override;
 
   TransactionLockMgr::LockStatusData GetLockStatusData() override;
+  struct CommitEntry {
```

Comment:
It is used by transactions when interacting with the db. But lets sleep on that. This is not a public API and we always can get back to it and do a refactoring.